frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/deleteBulkReleasePreview.jsp

```diff
@@ -0,0 +1,118 @@
+<%--
+  ~ Copyright Siemens AG, 2013-2017, 2019. Part of the SW360 Portal Project.
+  ~ With modifications by Bosch Software Innovations GmbH, 2016.
+  ~
+  ~ This program and the accompanying materials are made
+  ~ available under the terms of the Eclipse Public License 2.0
+  ~ which is available at https://www.eclipse.org/legal/epl-2.0/
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0
+--%>
```

Comment:
Update the license header.

Reply:
done

Reply:
There are four newly generated files, can you make the license header info uniform for all four.

Reply:
Ah ,  OK I will set a uniform header for all 4 files.